### PR TITLE
Manual updates 20240618

### PR DESCRIPTION
**Pull request by moljac**
### Describe your contribution

Updates and new dependencies

- `com.google.android.gms:playservices-ads` - 22.3.0 -> 22.4.0
- `com.google.android.gms:playservices-ads-base` - 22.3.0 -> 22.4.0
- `com.google.android.gms:playservices-ads-identifier` - 18.0.1 -> 18.1.0
- `com.google.android.gms:playservices-ads-lite` - 22.3.0 -> 22.4.0
- `com.google.android.gms:playservices-afs-native` - 19.0.3 -> 19.1.0
- `com.google.android.gms:playservices-analytics` - 18.0.4 -> 18.1.0
- `com.google.android.gms:playservices-analytics-impl` - 18.0.4 -> 18.1.0
- `com.google.android.gms:playservices-appindex` - 16.1.0 -> 16.2.0
- `com.google.android.gms:playservices-appset` - 16.0.2 -> 16.1.0
- `com.google.android.gms:playservices-auth` - 21.1.1 -> 21.2.0
- `com.google.android.gms:playservices-auth-api-phone` - 18.0.2 -> 18.1.0
- `com.google.android.gms:playservices-auth-base` - 18.0.12 -> 18.0.13
- `com.google.android.gms:playservices-awareness` - 19.0.1 -> 19.1.0
- `com.google.android.gms:playservices-base` - 18.4.0 -> 18.5.0
- `com.google.android.gms:playservices-basement` - 18.3.0 -> 18.4.0
- `com.google.android.gms:playservices-cast` - 21.4.0 -> 21.5.0
- `com.google.android.gms:playservices-cast-framework` - 21.4.0 -> 21.5.0
- `com.google.android.gms:playservices-cast-tv` - 21.0.1 -> 21.1.0
- `com.google.android.gms:playservices-cloud-messaging` - 17.2.0 -> 17.3.0
- `com.google.android.gms:playservices-cronet` - 18.0.1 -> 18.1.0
- `com.google.android.gms:playservices-device-performance` - 16.0.0 -> 16.1.0
- `com.google.android.gms:playservices-fido` - 21.0.0 -> 21.1.0
- `com.google.android.gms:playservices-fitness` - 21.1.0 -> 21.2.0
- `com.google.android.gms:playservices-flags` - 18.0.1 -> 18.1.0
- `com.google.android.gms:playservices-games` - 23.1.0 -> 23.2.0
- `com.google.android.gms:playservices-games-v2` - 20.0.0 -> 20.1.0
- `com.google.android.gms:playservices-identity` - 18.0.1 -> 18.1.0
- `com.google.android.gms:playservices-instant-apps` - 18.0.1 -> 18.1.0
- `com.google.android.gms:playservices-location` - 21.2.0 -> 21.3.0
- `com.google.android.gms:playservices-measurement` - 21.3.0 -> 21.4.0
- `com.google.android.gms:playservices-measurement-api` - 21.3.0 -> 21.4.0
- `com.google.android.gms:playservices-measurement-base` - 21.3.0 -> 21.4.0
- `com.google.android.gms:playservices-measurement-impl` - 21.3.0 -> 21.4.0
- `com.google.android.gms:playservices-measurement-sdk` - 21.3.0 -> 21.4.0
- `com.google.android.gms:playservices-measurement-sdk-api` - 21.3.0 -> 21.4.0
- `com.google.android.gms:playservices-nearby` - 19.2.0 -> 19.3.0
- `com.google.android.gms:playservices-oss-licenses` - 17.0.1 -> 17.1.0
- `com.google.android.gms:playservices-pal` - 20.2.0 -> 20.3.0
- `com.google.android.gms:playservices-password-complexity` - 18.0.1 -> 18.1.0
- `com.google.android.gms:playservices-pay` - 16.4.0 -> 16.5.0
- `com.google.android.gms:playservices-places` - 17.0.0 -> 17.1.0
- `com.google.android.gms:playservices-places-placereport` - 17.0.0 -> 17.1.0
- `com.google.android.gms:playservices-recaptcha` - 17.0.1 -> 17.1.0
- `com.google.android.gms:playservices-safetynet` - 18.0.1 -> 18.1.0
- `com.google.android.gms:playservices-stats` - 17.0.3 -> 17.1.0
- `com.google.android.gms:playservices-tagmanager` - 18.0.4 -> 18.1.0
- `com.google.android.gms:playservices-tagmanager-api` - 18.0.4 -> 18.1.0
- `com.google.android.gms:playservices-tagmanager-v4-impl` - 18.0.4 -> 18.1.0
- `com.google.android.gms:playservices-tasks` - 18.1.0 -> 18.2.0
- `com.google.android.gms:playservices-thread-network` - 16.0.0 -> 16.0.1
- `com.google.android.gms:playservices-wallet` - 19.3.0 -> 19.4.0
- `com.google.android.gms:playservices-wearable` - 18.1.0 -> 18.2.0
- `com.google.android.tv:tv-ads` -  -> 1.0.0
- `com.google.firebase:firebase-ads` - 22.3.0 -> 22.4.0
- `com.google.firebase:firebase-ads-lite` - 22.3.0 -> 22.4.0
- `com.google.firebase:firebase-analytics` - 21.3.0 -> 21.4.0
- `com.google.firebase:firebase-analytics-ktx` - 21.3.0 -> 21.4.0
- `com.google.firebase:firebase-crashlytics` - 19.0.0 -> 19.0.1
- `com.google.firebase:firebase-crashlytics-ktx` - 19.0.0 -> 19.0.1
- `com.google.firebase:firebase-crashlytics-ndk` - 19.0.0 -> 19.0.1
- `com.google.firebase:firebase-dynamic-links` - 22.0.0 -> 22.1.0
- `com.google.firebase:firebase-dynamic-links-ktx` - 22.0.0 -> 22.1.0
- `com.google.firebase:firebase-perf` - 21.0.0 -> 21.0.1
- `com.google.firebase:firebase-perf-ktx` - 21.0.0 -> 21.0.1
- `com.google.firebase:firebase-sessions` - 2.0.0 -> 2.0.1
- `com.google.android.libraries.places:places` - 3.4.0 -> 3.5.0
- `com.android.billingclient:billing` - 6.2.1 -> 7.0.0
- `com.android.billingclient:billing-ktx` - 6.2.1 -> 7.0.0
- `com.google.android.recaptcha:recaptcha` - 18.4.0 -> 18.5.1
- `com.google.dagger:dagger` - 2.50 -> 2.51.1
- `com.google.errorprone:error_prone_annotations` - 2.27.1 -> 2.28.0
- `com.google.protobuf:protobuf-javalite` - 4.26.1 -> 4.27.1
- `com.squareup.retrofit2:adapter-rxjava2` - 2.10.0 -> 2.11.0
- `com.squareup.retrofit2:converter-gson` - 2.10.0 -> 2.11.0
- `com.squareup.retrofit2:converter-scalars` - 2.10.0 -> 2.11.0
- `com.squareup.retrofit2:retrofit` - 2.10.0 -> 2.11.0
- `io.grpc:grpc-android` - 1.59.1 -> 1.60.2
- `io.grpc:grpc-api` - 1.59.1 -> 1.60.2
- `io.grpc:grpc-context` - 1.59.1 -> 1.60.2
- `io.grpc:grpc-core` - 1.59.1 -> 1.60.2
- `io.grpc:grpc-okhttp` - 1.59.1 -> 1.60.2
- `io.grpc:grpc-protobuf-lite` - 1.59.1 -> 1.60.2
- `io.grpc:grpc-stub` - 1.59.1 -> 1.60.2
- `io.grpc:grpc-util` - 1.59.1 -> 1.60.2
- `com.google.inject:guice` - 6.0.0 -> 7.0.0
- `org.ow2.asm:asm` - 9.5 -> 9.6
- `com.google.ads.interactivemedia.v3:interactivemedia` - 3.29.0 -> 3.30.0
- `androidx.credentials:credentials-playservices-auth` -  -> 1.2.2
- `androidx.privacysandbox.ads:ads-adservices` -  -> 1.0.0beta05
- `androidx.privacysandbox.ads:ads-adservices-java` -  -> 1.0.0beta05

### CODE REVIEW

**Comment by moljac** [comment]
aapt errors 

```
./generated/com.google.firebase.firebase-auth/obj/Debug/net8.0-android/lp/85/jl/res/drawable-watch-v20/common_google_signin_btn_text_dark_normal.xml(2): error APT2260: resource drawable/googleg_standard_color_18 (aka Xamarin.Firebase.Auth:drawable/googleg_standard_color_18) not found. [./generated/com.google.firebase.firebase-auth/com.google.firebase.firebase-auth.csproj::TargetFramework=net8.0-android]
./generated/com.google.firebase.firebase-auth/obj/Debug/net8.0-android/lp/85/jl/res/drawable-watch-v20/common_google_signin_btn_text_dark_normal.xml(2): error APT2260:  [./generated/com.google.firebase.firebase-auth/com.google.firebase.firebase-auth.csproj::TargetFramework=net8.0-android]
./generated/com.google.firebase.firebase-auth/obj/Debug/net8.0-android/lp/85/jl/res/drawable-watch-v20/common_google_signin_btn_text_disabled.xml(2): error APT2260: resource drawable/googleg_disabled_color_18 (aka Xamarin.Firebase.Auth:drawable/googleg_disabled_color_18) not found. [./generated/com.google.firebase.firebase-auth/com.google.firebase.firebase-auth.csproj::TargetFramework=net8.0-android]
./generated/com.google.firebase.firebase-auth/obj/Debug/net8.0-android/lp/85/jl/res/drawable-watch-v20/common_google_signin_btn_text_disabled.xml(2): error APT2260:  [./generated/com.google.firebase.firebase-auth/com.google.firebase.firebase-auth.csproj::TargetFramework=net8.0-android]
./generated/com.google.firebase.firebase-auth/obj/Debug/net8.0-android/lp/85/jl/res/drawable-watch-v20/common_google_signin_btn_text_light_normal.xml(2): error APT2260: resource drawable/googleg_standard_color_18 (aka Xamarin.Firebase.Auth:drawable/googleg_standard_color_18) not found. [./generated/com.google.firebase.firebase-auth/com.google.firebase.firebase-auth.csproj::TargetFramework=net8.0-android]
./generated/com.google.firebase.firebase-auth/obj/Debug/net8.0-android/lp/85/jl/res/drawable-watch-v20/common_google_signin_btn_text_light_normal.xml(2): error APT2260:  [./generated/com.google.firebase.firebase-auth/com.google.firebase.firebase-auth.csproj::TargetFramework=net8.0-android]
/usr/local/share/dotnet/packs/Microsoft.Android.Sdk.Darwin/34.0.113/tools/Xamarin.Android.Aapt2.targets(156,3): error APT2061: failed linking file resources. [./generated/com.google.firebase.firebase-auth/com.google.firebase.firebase-auth.csproj::TargetFramework=net8.0-android]
/usr/local/share/dotnet/packs/Microsoft.Android.Sdk.Darwin/34.0.113/tools/Xamarin.Android.Aapt2.targets(156,3): error APT2061:  [./generated/com.google.firebase.firebase-auth/com.google.firebase.firebase-auth.csproj::TargetFramework=net8.0-android]
    503 Warning(s)
    4 Error(s)
```

for 

```xml
<?xml version='1.0' encoding='UTF-8'?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <modelVersion>4.0.0</modelVersion>
  <groupId>com.google.firebase</groupId>
  <artifactId>firebase-auth</artifactId>
  <version>23.0.0</version>
  <packaging>aar</packaging>
  <dependencies>
    <dependency>
      <groupId>androidx.browser</groupId>
      <artifactId>browser</artifactId>
      <version>1.4.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>androidx.collection</groupId>
      <artifactId>collection</artifactId>
      <version>1.0.0</version>
      <scope>compile</scope>
      <type>jar</type>
    </dependency>
    <dependency>
      <groupId>androidx.credentials</groupId>
      <artifactId>credentials</artifactId>
      <version>1.2.0-rc01</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>androidx.credentials</groupId>
      <artifactId>credentials-play-services-auth</artifactId>
      <version>1.2.0-rc01</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>androidx.fragment</groupId>
      <artifactId>fragment</artifactId>
      <version>1.1.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>androidx.localbroadcastmanager</groupId>
      <artifactId>localbroadcastmanager</artifactId>
      <version>1.0.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-auth-api-phone</artifactId>
      <version>18.0.2</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-basement</artifactId>
      <version>18.3.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.gms</groupId>
      <artifactId>play-services-tasks</artifactId>
      <version>18.1.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.play</groupId>
      <artifactId>integrity</artifactId>
      <version>1.2.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.android.recaptcha</groupId>
      <artifactId>recaptcha</artifactId>
      <version>18.4.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.firebase</groupId>
      <artifactId>firebase-annotations</artifactId>
      <version>16.2.0</version>
      <scope>compile</scope>
      <type>jar</type>
    </dependency>
    <dependency>
      <groupId>com.google.firebase</groupId>
      <artifactId>firebase-appcheck-interop</artifactId>
      <version>17.0.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.firebase</groupId>
      <artifactId>firebase-auth-interop</artifactId>
      <version>20.0.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.firebase</groupId>
      <artifactId>firebase-common</artifactId>
      <version>21.0.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.firebase</groupId>
      <artifactId>firebase-common-ktx</artifactId>
      <version>21.0.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>com.google.firebase</groupId>
      <artifactId>firebase-components</artifactId>
      <version>18.0.0</version>
      <scope>compile</scope>
      <type>aar</type>
    </dependency>
    <dependency>
      <groupId>org.jetbrains.kotlin</groupId>
      <artifactId>kotlin-stdlib</artifactId>
      <version>1.7.10</version>
      <scope>compile</scope>
      <type>jar</type>
    </dependency>
  </dependencies>
  <name>firebase-auth</name>
  <licenses>
    <license>
      <name>Android Software Development Kit License</name>
      <url>https://developer.android.com/studio/terms.html</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
</project>
```

**Comment by moljac** [comment]
downgrading `com.google.firebase:firebase-auth:23.0.0` to `com.google.firebase:firebase-auth:22.3.1` for now

**Comment by moljac** [comment]
```
find externals -type f -iname common_google_signin_btn_text_dark_normal.xml
```

results in:

```
externals/com.google.android.gms/play-services-base/res/drawable/common_google_signin_btn_text_dark_normal.xml
externals/com.google.android.gms/play-services-auth/res/drawable-watch-v20/common_google_signin_btn_text_dark_normal.xml
```

**Comment by moljac** [comment]
ACW errors

```
MSBUILD : java error JAVA0000: Error in obj/Debug/net8.0-android/lp/548/jl/classes.jar:com/google/protobuf/DescriptorProtos$1.class: [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: Type com.google.protobuf.DescriptorProtos$1 is defined multiple times: obj/Debug/net8.0-android/lp/548/jl/classes.jar:com/google/protobuf/DescriptorProtos$1.class, obj/Debug/net8.0-android/lp/405/jl/libs/8E428AF64A795598.jar:com/google/protobuf/DescriptorProtos$1.class [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: Compilation failed [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: java.lang.RuntimeException: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: obj/Debug/net8.0-android/lp/548/jl/classes.jar [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: com/google/protobuf/DescriptorProtos$1.class [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:135) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.D8.main(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:5) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: Caused by: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: obj/Debug/net8.0-android/lp/548/jl/classes.jar:com/google/protobuf/DescriptorProtos$1.class [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at Version.fakeStackEntry(Version_8.2.33.java:0) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.T.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:5) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:82) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:32) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:31) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.utils.S0.b(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:2) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:42) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.D8.b(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:13) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:40) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:122) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	... 1 more [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: Caused by: com.android.tools.r8.utils.b: Type com.google.protobuf.DescriptorProtos$1 is defined multiple times: obj/Debug/net8.0-android/lp/548/jl/classes.jar:com/google/protobuf/DescriptorProtos$1.class, obj/Debug/net8.0-android/lp/405/jl/libs/8E428AF64A795598.jar:com/google/protobuf/DescriptorProtos$1.class [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.utils.Q2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:21) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.utils.D2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:54) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.utils.D2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:10) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at java.base/java.util.concurrent.ConcurrentHashMap.merge(ConcurrentHashMap.java:2048) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.utils.D2.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:6) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.graph.m4$a.d(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:6) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:61) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:12) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:9) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:45) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.D8.d(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:17) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.D8.c(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:69) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	at com.android.tools.r8.utils.S0.a(R8_8.2.33_429c93fd24a535127db6f4e2628eb18f2f978e02f99f55740728d6b22bef16dd:28) [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: 	... 6 more [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
MSBUILD : java error JAVA0000: Directory 'obj/Debug/net8.0-android/lp/548' is from 'protolitewellknowntypes-18.0.0.aar'. [./samples/dotnet/BuildAllDotNet/BuildAllDotNet.csproj::TargetFramework=net8.0-android]
    15 Warning(s)
    1 Error(s)
```

File: config.json [hunk 1]
```diff
@@ -124,24 +124,24 @@
...
         "nugetId": "Xamarin.GooglePlayServices.Auth.Api.Phone",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-auth-base",
-        "version": "18.0.12",
-        "nugetVersion": "118.0.12",
+        "version": "18.0.13",
+        "nugetVersion": "18.0.13",
```
Comment (jpobst) [inline config.json:144]: Should be `118.0.13`.

File: config.json [hunk 2]
```diff
@@ -124,24 +124,24 @@
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-auth",
-        "version": "21.1.1",
-        "nugetVersion": "121.1.1",
+        "version": "21.2.0",
+        "nugetVersion": "21.2.0",
```
Comment (jpobst) [inline config.json:128]: Should be `121.2.0`.

File: config.json [hunk 3]
```diff
@@ -156,48 +156,48 @@
...
         "nugetId": "Xamarin.GooglePlayServices.Awareness",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-base",
-        "version": "18.4.0",
-        "nugetVersion": "118.4.0",
+        "version": "18.5.0",
+        "nugetVersion": "18.5.0",
```
Comment (jpobst) [inline config.json:168]: Should be `118.5.0`, there are more than I have marked, they should all be fixed.

**Review by jpobst — APPROVED** [review]

**Pull request merged**